main.js

```diff
@@ -45,3 +70,24 @@ $customizeForm.addEventListener('submit', function (event) {
   jerseyConfig.number = $number.value
   changeJersey(jerseyConfig)
 })
+
+$cartButton.addEventListener('click', function (event) {
+  if (validate(jerseyConfig)) {
+    jerseyConfig.quantity = parseInt($quantity.value, 10)
+    cart.push(jerseyConfig)
+    cartCounter += jerseyConfig.quantity
+    $cartCounter.textContent = cartCounter
+    $customizeForm.reset()
+    resetJersey()
+  }
+})
```

Comment:
you want to create an `addToCart(jerseyConfig, qty)` function here, and start modeling your cart. Your cart will probably be a list of jersey configs. It should add a new jersey config to that array, and so the cart counter will simply be `cart.length`, and you no longer rely on the `cartCounter`